### ISSUE TITLE
Advance numpy to 1.16.0

### ISSUE THREAD

**Issue by cwhanse** [pvlib/pvlib-python#1399]
The py36-min environment pins numpy at 1.12.0. #717 (bifacial.infinite_shdes) uses `numpy.linspace` with non-scalar start and stop values, a capability introduced with numpy 1.16.

**Describe the solution you'd like**
Advance numpy to 1.16 in py36-min

**Describe alternatives you've considered**
- awkward code in #717 that checks numpy version and executes a loop if numpy < 1.16
- deploying a function that fails tests in py36-min
- requiring numpy >= 1.16 for the bifacial.infinite_sheds functions

**Additional context**
If we decide to advance numpy, is a deprecation period needed?

**Comment by wholmgren** [comment]
Yes, definitely fine to advance to 1.16. If we aggressively followed NEP-29, we would be at python 3.8+ and numpy 1.19+.

No deprecation needed.

https://numpy.org/neps/nep-0029-deprecation_policy.html
